examples/fabric-bridge-app/linux/RpcClient.cpp

```diff
@@ -73,8 +74,31 @@ CHIP_ERROR OpenCommissioningWindow(NodeId nodeId)
         return CHIP_ERROR_BUSY;
     }
 
-    chip_rpc_DeviceInfo device;
-    device.node_id = nodeId;
+    chip_rpc_DeviceCommissioningWindowInfo device;
+    device.node_id               = nodeId;
+    device.commissioning_timeout = commissioningTimeout;
+    device.discriminator         = discriminator;
+    device.iterations            = iterations;
+
+    if (salt.HasValue())
+    {
+        if (salt.Value().size() > sizeof(device.salt.bytes))
+        {
+            return CHIP_ERROR_INTERNAL;
```

Comment:
insufficient memory or space seems like a better error code than internal.

use CHIP_ERROR_NO_MEMORY or CHIP_ERROR_BUFFER_TOO_SMALL or similar. (I like buffer to small for now)

Reply:
Addressed in https://github.com/project-chip/connectedhomeip/pull/34209/commits/d29e4fc464816478013110707686ad8d75a5d806